src/BsReactstrap__Alert.re

```diff
@@ -1,44 +1,57 @@
 [@bs.module "reactstrap"] external alert : ReasonReact.reactClass = "Alert";
 
-[@bs.obj]
-external makeProps : (
-  ~className: string=?,
-  ~closeClassName: string=?,
-  ~closeAriaLabel: string=?,
-  ~cssModule: 'a=?,
-  ~color: string=?,
-  ~isOpen: bool=?,
-  ~toggle: 'b=?,
-  ~tag: 'c=?,
-  ~transition: 'd=?,
-  unit
-) => _ = "";
+type toggle;
+type transition;
 
-let make = (
-  ~className=?,
-  ~closeClassName=?,
-  ~closeAriaLabel=?,
-  ~cssModule=?,
-  ~color=?,
-  ~isOpen=?,
-  ~toggle=?,
-  ~tag=?,
-  ~transition=?,
-  children
-) =>
+[@bs.deriving abstract]
+type props = {
+  [@bs.optional]
```

Comment:
Ugh, why `refmt` is doing that? This would've been much more readable:
```
[@bs.deriving abstract]
type props = {
  [@bs.optional] body: bool,
  [@bs.optional] bottom: bool,
  [@bs.optional] className: string,
}
```

Reply:
I have no idea. I love Reason, but my number 1 complaint is the lack of configuration for some stuff (namespace, refmt). But yes I have no control over this unfortunately.

Reply:
Totally agree